scripts/util.py

```diff
@@ -152,6 +155,61 @@
     None
 ] = lambda self, data: yaml.representer.SafeRepresenter.represent_str(self, str(data))
 
+with open(PROVIDERS_FILE) as stream:
+    try:
+        providers = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        providers = None
+        print(exc)
+prv = NSDict(providers)
+
+universe_domain = "googleapis.com"
+universe_sa_credentials = None
+endpoint_versions = None
```

Comment:
Having it set to `{}` by default will save you few lines of code.

Reply:
Obsolete.